easybuild/easyconfigs/c/Clang/Clang-3.6.0-GCC-4.8.4.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = "Clang"
+version = "3.6.0"
+
+homepage = "http://clang.llvm.org/"
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCC', 'version': '4.8.4'}
```

Comment:
gcc/4.8.4 is about staying in alignment with #1294, 10 reasons stated just there on the why of this.